recipes-security/swm/smw_git.bb

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020-23 NXP
```

Comment:
Is this recipe from a NXP BSP? Could you, please, add in the commit log which tag/version exactly?